### PR TITLE
Revert most of changes by 3651

### LINKED ISSUE #3712: Revert changes introduced by PR#3651 after next release of MLFlow
### Motivation

https://github.com/optuna/optuna/pull/3651 has introduced chunking to send too long tag/metric/params to avoid mlflow's API limit. Hopefully, mlflow itself will handle this case by https://github.com/mlflow/mlflow/pull/6052, so Optuna doesn't need the logic introduced by #3651 with the next mlflow version anymore.

### Description

Revert all changes introduced by #3651 after the next mlflow release that includes https://github.com/mlflow/mlflow/pull/6052.

### Alternatives (optional)

_No response_

### Additional context (optional)

_No response_

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

Resolve #3712 because the introduced changes by https://github.com/optuna/optuna/pull/3651 are unnecessary anymore, thanks to the latest stable mlflow (concretely, https://github.com/mlflow/mlflow/pull/6052).

## Description of the changes

Revert the changes by https://github.com/optuna/optuna/pull/3651 except for https://github.com/optuna/optuna/pull/3651/files#diff-88ddb9fb7f05a37d5fc720d442a748b3189ebf4f43500e9709fd6507803c2e30R168, which was a sort of code-fix.

### CODE REVIEW

**Comment by HideakiImamura** [comment]
@contramundum53 @xadrianzetx Could you review this PR since you were reviewers of #3651?

File: optuna/integration/mlflow.py [hunk 1]
```diff
@@ -281,9 +277,7 @@ def _set_tags(self, trial: optuna.trial.FrozenTrial, study: optuna.study.Study)
                 tags[key] = "{}...".format(value[: max_val_length - 3])
 
         # This sets the tags for MLflow.
```
Comment (xadrianzetx) [inline optuna/integration/mlflow.py:279]: Could we also remove this comment? Seems unnecessary.

Comment (nzw0301) [reply optuna/integration/mlflow.py:279]: Nice point. I agree with you.

**Review by xadrianzetx — APPROVED** [review]
Looks great!

**Review by contramundum53 — APPROVED** [review]
LGTM!

**Pull request merged**